ClassesHomework/Cat.cs

```diff
@@ -6,10 +6,59 @@
 
 namespace ClassesHomework
 {
-    // Write your Cat class here
-    //class Cat
-    //{
-        
-    //}
+    class Cat : Animal
+    {
+        public Cat()
+        {
+            IsSitting = false;
+            Age = 0;
+
+        }
+
+        public string Breed { get; private set; }
+        public int Age { get; private set; }
+        public bool IsSitting { get; private set; }
+
+        public Cat(string breed)
+        {
+            Breed = breed;
+        }
+
+        public Cat(int age)
+        {
+            Age = age;
+        }
+
+        public Cat(string breed, int age)
+        {
+            Breed = breed;
+            Age = age;
+        }
+
+        public string Speak()
```

Comment:
This is a place where the difference between _hiding_ and _overriding_ is important. Cat.Speak() is hiding Animal.Speak().

This means when a Cat object is accessed through a Cat variable, it speaks like a cat. But when a  Cat object is accessed through an Animal variable, it speaks like an Animal, not a Cat.

Take a look at the warning here. It tells what keyword you need to add.